File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    
    do
    {
        //ask user height of half-pyramide
        printf("Height: ");
        height = get_int();
    }
    while( height < 0 || height > 23);
    
    for (int i = 0; i < height; i++)
    {   
        int row = i+1;
        int spaces = height;
        int hashes = 0;
        
        //print spaces
        while(spaces > row)
        {
            printf(" ");
            spaces--;
        }

        //print hashes
        while(hashes < row + 1)
        {
            printf("#");
            hashes++;
        }

        //print \n
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

# Check for proper usage
while True:
    height = get_int("height: ")
    if height >= 0 and height < 23:
        break
    print("Please non-negative and not more than 23")

# iterate row
for row in range(height):
    # print cell in row
    for count in range(2):
        if count == 0:
            print(" " * (height - row - 1), end = "")
            print("#" * (row + 1), end = "")
            print("  ", end="")
        # second tower but mirrow
        else:
            print("#" * (row + 1), end = "")
    # print new line
    print()